src/test/java/org/opensearch/knn/plugin/action/RestTrainModelHandlerIT.java

```diff
@@ -192,6 +193,68 @@ public void testTrainModel_fail_tooMuchData() throws Exception {
         assertTrainingFails(modelId, 30, 1000);
     }
 
+    public void testTrainModel_fail_commaInDescription() throws Exception {
+        // Test checks that training when passing in an id succeeds
+
+        String modelId = "test-model-id";
+        String trainingIndexName = "train-index";
+        String trainingFieldName = "train-field";
+        int dimension = 8;
+
+        // Create a training index and randomly ingest data into it
+        createBasicKnnIndex(trainingIndexName, trainingFieldName, dimension);
+        int trainingDataCount = 200;
+        bulkIngestRandomVectors(trainingIndexName, trainingFieldName, trainingDataCount, dimension);
```

Comment:
do we need to ingest vectors for this test? Training should fail because of the comma in model description.

Reply:
Good catch